modules/apps/foundation/login/login-web/src/main/resources/META-INF/resources/navigation.jsp

```diff
@@ -17,41 +17,47 @@
 <%@ include file="/init.jsp" %>
 
 <liferay-util:buffer var="navigation">
-	<liferay-ui:icon-list>
-		<liferay-util:dynamic-include key="com.liferay.login.web#/navigation.jsp#pre" />
+	<liferay-util:dynamic-include key="com.liferay.login.web#/navigation.jsp#pre" />
 
-		<%
-		for (String section : PropsValues.LOGIN_FORM_NAVIGATION_PRE) {
-		%>
+	<%
+	for (String section : PropsValues.LOGIN_FORM_NAVIGATION_PRE) {
+	%>
 
-			<liferay-util:include page='<%= "/navigation/" + _getSectionJsp(section) + ".jsp" %>' portletId="<%= portletDisplay.getRootPortletId() %>" servletContext="<%= application %>" />
+		<liferay-util:include page='<%= "/navigation/" + _getSectionJsp(section) + ".jsp" %>' portletId="<%= portletDisplay.getRootPortletId() %>" servletContext="<%= application %>" />
 
-		<%
-		}
+	<%
+	}
 
-		for (String section : PropsValues.LOGIN_FORM_NAVIGATION_POST) {
-		%>
+	for (String section : PropsValues.LOGIN_FORM_NAVIGATION_POST) {
+	%>
 
-			<liferay-util:include page='<%= "/navigation/" + _getSectionJsp(section) + ".jsp" %>' portletId="<%= portletDisplay.getRootPortletId() %>" servletContext="<%= application %>" />
+		<liferay-util:include page='<%= "/navigation/" + _getSectionJsp(section) + ".jsp" %>' portletId="<%= portletDisplay.getRootPortletId() %>" servletContext="<%= application %>" />
 
-		<%
-		}
-		%>
+	<%
+	}
+	%>
 
-		<liferay-util:dynamic-include key="com.liferay.login.web#/navigation.jsp#post" />
-	</liferay-ui:icon-list>
+	<liferay-util:dynamic-include key="com.liferay.login.web#/navigation.jsp#post" />
 </liferay-util:buffer>
 
 <%
 navigation = navigation.trim();
 %>
 
 <c:if test="<%= Validator.isNotNull(navigation) %>">
-	<div class="navigation">
+	<ul class="lfr-nav nav nav-tabs nav-tabs-default navigation" id="<portlet:namespace />navigation">
 		<%= navigation %>
-	</div>
+	</ul>
 </c:if>
 
+<aui:script>
+	var navigation = $('#<portlet:namespace />navigation');
```

Comment:
We should only refer to jQuery with `$` (or lodash with `_`) in an `aui:script` block that has the attribute `sandbox="<%= true %>"`, otherwise, we need to refer to `AUI.$`.
The reason is so that if someone wants to load a different version of jQuery, they can without breaking our plugins.